astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,71 @@
+
+{/* more */}
+
+There are many different ways to implement authentication, but the most common approach is to use a username and password. Yep, just like back in 1960 when [The Everly Brothers](https://youtu.be/KtY3L5tDN_E?si=BATB3BzJJFpWsHmB) had one of the top songs out there, this method is still most popular today. Just as the Everly Brothers' time at the top of the charts has passed, so has the time for passwords. I don't want to hear about them any more!
+
+A more modern approach is to use a single sign-on (SSO) solution. SSO solutions allow users to log in to multiple applications using a single set of credentials. This can be convenient for users, as they do not have to remember multiple usernames and passwords. This method came about around the same time Eminem hit the [billboard top 100](https://youtu.be/6mPCy9LMxTI?si=5A2I-N1neb_8rGOl) for the first time. SSO often has the problem of including the first login requirement of using a username and password still to this day. Even Eminem made some changes and remains relevant!
+
+My point is that Authentication is an important security measure, but it can also be complex to implement. This is why we have been hanging on to these old methods so long! When you are creating a new application (unless it is one for Authentication) the last thing you care about is trying to roll your own. There are many different authentication methods to choose from, and each method has its own advantages and disadvantages. Additionally, authentication needs to be implemented in a secure way to prevent attackers from gaining unauthorized access to applications. All I care about is that my data is secure when creating the next [Napster](https://youtu.be/CKrdsGdLVQ8?si=bTp4DHKS4tlpQzfW) so everyone can share their 1990's music with me again!
+
+![Napster app from Wikipedia](/img/blogs/i-just-want-authentication-to-work/napster.png)
+
+## The benefits of using an authentication service
+
+There are many benefits to using an authentication service. Authentication services can help you to:
+
+* Save time and money: Implementing and maintaining authentication can be time-consuming and expensive. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Improve security: Authentication services can help you to improve the security of your applications by using best practices and the latest security technologies keeping up to date with the latest security patches.
+* Reduce the risk of compliance violations: Authentication services can help you to meet compliance requirements by providing features such as two-factor authentication and user provisioning.
+
+The last thing that I want is a call on Saturday night that someone hacked our password database that we built in house and didn't secure!
```

Comment:
```suggestion
The last thing that I want is a call on Saturday night that someone hacked our password database that we built in house two years ago and didn't maintain securely!
```

This feels like a realer situation to me. If it doesn't address your point, no worries.